packages/api/src/schema/api-schema.yaml

```diff
@@ -1154,8 +1185,70 @@ components:
         name:
           type: string
           description: |
-            Name of the asset. This is not necessarily the filename, can be a
-            custom name or title
+            The name of the asset. This is not necessarily the filename - it can be a
+            custom name or title.
+          example: filename.mp4
+        staticMp4:
+          type: boolean
+          description: Whether to generate MP4s for the asset.
+          example: true
+        playbackPolicy:
+          $ref: "#/components/schemas/playback-policy"
+        creatorId:
+          $ref: "#/components/schemas/input-creator-id"
+        storage:
```

Comment:
Is this the same field as `new-asset-from-url-payload`? If so, we should avoid repeating it and use a `$ref` instead (maybe `new-asset-from-url-payload` references this one, since it's the more specific one)